### DISCUSSION TITLE
Need help to report issues on githubstatus.com

### DISCUSSION THREAD

**Discussion by rahulbohra** [community/community#23019] [New to GitHub]
<p>Hi Team,</p>
<p>Please guide me where can I report issues of <a href="http://githubstatus.com" rel="nofollow noopener">githubstatus.com</a></p>
<p>Thanks,</p>
<p>Rahul</p>

**Comment by nyahbhinghiPrincess** [comment]
<p>Hi <a class="mention" href="/rahulbohra">@rahulbohra</a> ,</p>
<p>This post was moved to a different board that fits your topic of discussion a bit better. This means you’ll get better engagement on your post, and it keeps our community organized so users can more easily find information.</p>
<p>As you’ll notice, your topic is now here in How to use Git and GitHub board. No action is needed on your part; you can continue the conversation as normal here.</p>
<p>Cheers!</p>

**Comment by ThirdEyeRose** [comment, marked as answer]
<p>Hi <a class="mention" href="/rahulbohra">@rahulbohra</a>,</p>
<p>The <a href="https://www.githubstatus.com/" rel="nofollow noopener">GitHub Status Page</a> is used by GitHub to inform users of ongoing incidents. You cannot report issues directly to it. If you are running into issues with GitHub, <a href="https://github.com/contact" rel="nofollow noopener">please contact support directly</a> as they will escalate issues that might need the attention of Engineering.</p>
<p>Thanks!</p>
